### PR TITLE
Mcp 180 jf change uploadpage display

### PR DESCRIPTION
**Pull request by jmfadeley**
### Related tickets:
MCP-180

### What's changing? Any breaking changes?
Basically, this is the heart of the application. 

### Manual test cases?
1) Run `yarn test`. The tests at the moment cover all the front end components I've built. A different ticket will handle the DataService test that is required.
2) Start the service with `yarn start`. Ensure that `mcp-data-mapping-util` is running via `flask run`. If you've set the backend's port to be different, update `REACT_APP_MCP_DATA_MAPPING_UTIL` in `.env` in this project to reflect this. 
3) Upload a few files. Courtesy of @TylerNoblett, I've included basic files in `/sample-data` that will cover the seven basic error types as well as the properly validating test cases. 

### Any additional context/background?
Needs a Dockerfile, needs a test for Dataservice, and to correct a function name for Grant. Then we're probably good to go on delivery. 

### Screenshots (if appropriate):
-

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,4 @@
 REACT_APP_GRAPHQL_API_URL=http://localhost:4000/
+REACT_APP_MCP_DATA_MAPPING_UTIL=http://localhost:5000/
+
+# My only secret... is that rainbows make me cry. T_T
```
Comment (TylerNoblett) [inline .env:4]: lol

File: src/components/UploadPage/UploadPage.test.tsx [hunk 2]
```diff
@@ -70,23 +64,29 @@ describe("Component: UploadPage", () => {
...
+    expect(errorPanel.get(1).key).toEqual("upload-page__file-report-1");
+
+    expect(wrapper.find(UploadFileReport).get(0).props).toEqual({
+      filename: "exampleFileNameA",
+      errorReports: MockUploadFileReport.exampleFileNameA
+    });
+    expect(wrapper.find(UploadFileReport).get(1).props).toEqual({
+      filename: "exampleFileNameB",
+      errorReports: MockUploadFileReport.exampleFileNameB
+    });
```
Comment (TylerNoblett) [inline src/components/UploadPage/UploadPage.test.tsx:89]: I've never done frontend testing before - this look really interesting, but generally makes sense when referencing the component

File: src/components/UploadPage/UploadPage.tsx [hunk 3]
```diff
@@ -1,112 +1,108 @@
...
-  [key: string]: any;
+  fileReports: FileReport;
 }
 
-class UploadPage extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = {
+const UploadPage: React.FC = () => {
+  const [uploadPageState, setUploadPageState] = React.useState<UploadPageState>(
```
Comment (TylerNoblett) [inline src/components/UploadPage/UploadPage.tsx:14]: love the fact that you're using hooks!

File: src/models/FileReport.tsx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    {
+      error_type: "incorrectly_formatted_dbq_id",
+      formatted_dbq_id: "Formatted DBQ",
+      object_id: "IncorrectlyFormattedDBQId",
+      message: "There aren't anymore monkeys! That's the whole barrel!"
+    },
+    {
+      error_type: "no_dbq_logical_combo",
+      object_id: "NoDBQLogicalErrorCombo",
+      message: "This is falling with style!"
```
Comment (TylerNoblett) [inline src/models/FileReport.tsx:157]: 😂these messages

**Review by TylerNoblett — APPROVED** [review]

File: src/components/UploadPage/UploadPage.test.tsx (hunk 2 above)
Comment (jmfadeley) [reply src/components/UploadPage/UploadPage.test.tsx:89]: This is Enzyme at work. I'm scheduled to do a dev review on it next month.

File: src/models/FileReport.tsx (hunk 4 above)
Comment (jmfadeley) [reply src/models/FileReport.tsx:157]: _Toy Story_ for the win.

**Review by puckybreg — APPROVED** [review]
🍖

**Pull request merged**